src/System.IO.Compression/src/System.IO.Compression.csproj

```diff
@@ -8,7 +8,7 @@
     <OutputType>Library</OutputType>
     <ProjectGuid>{5471BFE8-8071-466F-838E-5ADAA779E742}</ProjectGuid>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <UseECMAKey>true</UseECMAKey>
+    <UseECMAKey Condition="'$(UseECMAKey)'==''">true</UseECMAKey>
```

Comment:
Yes the EMCA key is generally special as it is supposed to be used by libraries that are part of the CLI spec. However it did get used by other assemblies such as this one and so the reason we are using it here is to maintain the same assembly identity.